go/exorelay/examples/index.go

```diff
@@ -0,0 +1,23 @@
+package exorelay_examples
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Originate/exocom/go/exorelay"
+	"github.com/Originate/exocom/go/structs"
+)
+
+type Example interface {
+	GetReceivedMessages() []structs.Message
+	Setup(exoRelay *exorelay.ExoRelay)
+}
+
+func Get(name string) Example {
+	switch name {
+	case "message-handler":
+		return &MessageHandlerExample{}
```

Comment:
I'd also like to move the sending feature into an example to make it easier to see